src/appshell/view/preferences/folderspreferencesmodel.h

```diff
@@ -41,6 +42,7 @@ class FoldersPreferencesModel : public QAbstractListModel, public async::Asyncab
     INJECT(appshell, notation::INotationConfiguration, notationConfiguration)
     INJECT(appshell, plugins::IPluginsConfiguration, pluginsConfiguration)
     INJECT(appshell, audio::IAudioConfiguration, audioConfiguration)
+    INJECT(appshell, vst::IVstConfiguration, vstConfiguration)
```

Comment:
Please, add a stub for this configuration. Otherwise, the app will crash if VST module is disabled  (it is disabled for everyone by default)